documentation/components/slider.html

```diff
@@ -0,0 +1,212 @@
+                                &lt;input type="number" class=" form-field current-value" id="current-value" value="500" min="0" max="1000">
+                            &lt;/div>
+                        </code>
+                    </pre>
+                </div>
+                <h4 class="my-2">JS</h4>
+
+                <!-- Simple slider example js code-->
+                <div class="code-container">
+                    <pre>
+                        <code class="language-js">
+                            const rangeSlider = document.querySelector(".range-slider")
+                            const currentValue = document.querySelector(".current-value")
+
+                            rangeSlider.addEventListener("input", () => {
+                                currentValue.value = rangeSlider.value
+                            })
+
+                            currentValue.addEventListener('input', () => {
+                                rangeSlider.value = currentValue.value
+                            })
+                        </code>
+                    </pre>
+                </div>
+            </section>
+        </main>
+        <!-- Main Content Section ends here-->
+
+        <!-- Table of Content section starts here-->
+        <section class="table-of-content-section text-gray text-sm mx-4">
+            <p class="toc-title py-2 pr-4 fw-semi-bold">ON THIS PAGE</p>
+            <nav class="ml-4">
+                <ul>
+                    <li>
+                        <a href="#simple-slider">Simple Slider</a>
+                    </li>
+                </ul>
+            </nav>
+        </section>
+        <!-- Table of Content section ends here-->
+    
+    </div>
+    <!-- Page Container Grid ends here-->
+
+    <!-- Footer starts here-->
+    <footer class="footer-container">
+        <section>
+            <p>Designed and built with <i class="fas fa-heart text-danger mx-1"></i> by <span class="ml-1">&lt;>Shaik Saadullah&lt;/></span></p>
+            <div class="social-links mt-4 fs-3">
+                <a href="https://github.com/shaik-saad">
+                    <i class="fa-brands fa-github mx-2"></i>
+                </a>
+                <a href="https://twitter.com/shaiksaadullah">
+                    <i class="fa-brands fa-twitter mx-2"></i>
+                </a>
+                <a href="https://www.linkedin.com/in/shaik-saad/">
+                    <i class="fa-brands fa-linkedin mx-2"></i>
+                </a>
+            </div>
+            <small class=" text-gray mt-2">© 2022 | <span class="text-primary">SLEEK UI</span></small>
+        </section>    
+    </footer>
+    <!-- Footer ends here-->
+
+    <!-- Highlight.js Script Tags -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+    <script>hljs.highlightAll();</script>
+
+    <!-- Site script Tag-->
+    <script src="../doc-site.js"></script>
+</body>
+</html>
```

Comment:
You could add an empty line at the end of the file to avoid such warnings by Github.

Reply:
sure, will do it